Filters/exclusions.txt

```diff
@@ -253,7 +253,7 @@
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/946
 ||customer.io^
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/943
-||branch.io^
+branch.io
```

Comment:
Not a goog idea. It is popular, but only few apps are broken.

Reply:
This rule unblocks `branch.io` main domain, which I think it's fine, so in my opinion we can leave it as it is.

Reply:
okay